protocol/protobuf/pairing.proto

```diff
@@ -322,7 +323,7 @@ message BackedUpProfile {
   string display_name = 2;
   uint64 display_name_clock = 3;
   repeated SyncProfilePicture pictures = 4;
-  repeated SyncSocialLinkSetting social_link_settings = 5;
+  repeated SocialLink social_links = 5;
   repeated SyncEnsUsernameDetail ens_username_details = 6;
```

Comment:
This will cause a breaking change between versions. Older versions of `status-go` will expect position `5` to be of type `SyncSocialLinkSetting` and parse `SocialLink ` data as a `SyncSocialLinkSetting`.

Frustrating I know. The fix is to add a new entry on an unused position for type `repeated SocialLink`

Reply:
@Samyoul thanks for the heads-up, I know for that, but opted in for this approach cause it is not not used anywhere. Mobile app doesn't use it yet and we're implementing this for the desktop app for the first time now. Want to say since syncing among devices was initially (which was recently) implemented on the status-go side was not in use.

Reply:
Ah, in that case go ahead.